package.json

```diff
@@ -716,15 +716,16 @@
     "jest-css-modules": "^2.1.0",
     "jest-junit": "^12.0.0",
     "miragejs": "^0.1.40",
-    "react": "^16.5.0",
-    "react-dom": "^16.5.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
```

Comment:
Nope, not now. Eventually, yes, but react 17 is STRIPES-722 and that's separate from STRIPES-721.